latex_access/tests/test_settings.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import latex_access
+from latex_access import settings as settings_module
+
+try:
+    settings_module.settings['brailletable'] = 'nemeth'
+    settings_module.brailleTableToUse()
+    settings_module.settings['brailletable'] = 'ueb'
+    settings_module.brailleTableToUse()
+    incompatible = False
+except AttributeError:
+    incompatible = True
+
+class TestSettings(unittest.TestCase):
+    def test_load_settings(self):
+        """Tests that settings are read from a file."""
+        settings_module.settings = {}
+        settings_module.loadSettings('latex_access/tests/test_settings.txt')
+        self.assertEqual(settings_module.settings, {'brailledollars': 'true', 'speakdollars': 'true', 'capitalisation':'6dot'})
```

Comment:
This test does not provide full coverage for `loadSettings`. Stuff which is completely untested:
- No check that when non-existent file is given as an argument `False` is returned
- No check that when the settings are loaded successfully `True` is returned.

Some parts of this function are also tested implicitly but explicit tests would be better (consider to just check that the old settings which are saved in `setUp` have not changed):
- Test with a separate appropriately named file where all lines are either empty or start with a semicolon
- Test with a file where all lines contain spaces or just a single word.